scripts/config/cassandra.toml

```diff
@@ -0,0 +1,36 @@
+[cassandra-store]
+type = "cassandra"
+create_db_template = """
+CREATE KEYSPACE IF NOT EXISTS %s WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1}  AND durable_writes = true
+CREATE TABLE IF NOT EXISTS %s.metric_idx (
+    id text,
+    orgid int,
+    partition int,
+    name text,
+    metric text,
+    interval int,
+    unit text,
+    mtype text,
+    tags set<text>,
+    lastupdate int,
+    PRIMARY KEY (partition, id)
+) WITH compaction = {'class': 'SizeTieredCompactionStrategy'}
+    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}
+"""
+
+[cassandra-idx]
```

Comment:
people may use cassandra for the store and not the index, or vice versa, so putting both in the same file seems confusing. hence my original suggestion to have a file for each index plugin and each store plugin

Reply:
alright. i thought having everything in a single file would make it easier to change things from an admin point of view.